trunk/src/lsc/myloopdef.py

```diff
@@ -1809,37 +1811,42 @@ def get_list(epoch=None, _telescope='all', _filter='', _bad='', _name='', _id=''
         ll = ''
     return ll
 
-def get_standards(epoch, name, filters):
+def get_standards(epoch, name, filters, standard_name='all', match_by_site=False):
     epochs = process_epoch(epoch)
-    targetid = lsc.mysqldef.gettargetid(name, '', '', lsc.conn)
+
+    if standard_name == 'all':
+        std_join = 'JOIN targets AS targstd ON std.targetid = targstd.id'
+        is_standard = 'targstd.classificationid = 1'
+    else:
+        std_join = 'JOIN targetnames AS targstd ON std.targetid = targstd.targetid'
+        is_standard = 'targstd.name = "{}"'.format(standard_name)
+
+    if match_by_site:  # join on site, telescope class, and instrument type
+        tel_join = '''((photlco AS obj JOIN targetnames AS targobj ON obj.targetid = targobj.targetid)
+                                       JOIN telescopes AS telobj ON obj.telescopeid = telobj.id)
+                                       JOIN instruments AS instobj ON obj.instrumentid = instobj.id,
+                      ((photlco as std {})
```

Comment:
I think we should always have named format keywords in python string replacements.

Reply:
It would be even better if we upgraded to Python 3 and could use f-strings.